src/main/java/io/fusionauth/domain/event/EventType.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2020, FusionAuth, All Rights Reserved
+ * Copyright (c) 2018-2022, FusionAuth, All Rights Reserved
```

Comment:
Looks like we have the wrong copyright here, can you update this to the Apache license.